SI/Sem.02/Pract.02/task06.cpp

```diff
@@ -0,0 +1,22 @@
+// ConsoleApplication4.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int a, b, c;
```

Comment:
Re-visit this task.
- Shouldn't the discriminant be a separate variable?
- Do we always have roots?
- Do we always have a quadratic equation?